src/pages/Reservation.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReservationManager from '../component/managers/ReservationManager';
+import Layout from '../component/wrapper/Layout';
+
+const Reservation = () => (
+  <Layout preAnimation="fadeIn">
+    <div style={{ width: '100vw' }}>
```

Comment:
- [ ] If you can use classes for styling here, it would be nice, since it's not a good practice to use the `style` attribute.